frontend/src/components/Table/Table.tsx

```diff
@@ -498,7 +512,7 @@ const Table = <T extends object>(props: PropsWithChildren<TableProps<T>>): React
                 innerRef={filterFormRef as any}
               >
                 {actions => (
-                  <>
+                  <Form>
```

Comment:
How does this change the component? Does it add behaviour of some kind?

Reply:
This allows to submit the form by clicking ENTER